src/App.js

```diff
@@ -19,13 +20,24 @@ function App() {
   //   setSalsa(window.Salsa('my-public-client-key', {env: 'sandbox'}));
   // }
 
-  return (
+  /**
+   * Load & Initialize Salsa via Library
+   *
+   * This initialization is required only once for the application.
+   */
+  // const [salsa, setSalsa] = useState();
+  // useEffect( () => {
+  //  loadSalsaJs('my-public-client-key', {env: 'sandbox'})
+  //    .then((salsaJs)=>setSalsa(salsaJs));
+  // }, [salsa]);
```

Comment:
```suggestion
  // }, []);
```
Should not need to execute more than on mount, right?